samcli/local/docker/manager.py

```diff
@@ -152,6 +157,21 @@ def has_image(self, image_name):
         except docker.errors.ImageNotFound:
             return False
 
+    def _is_rapid_image(self, image_name):
+        """
+        Is the image tagged as a RAPID clone?
+
+        : param string image_name: Name of the image
+        : return bool: True, if the image name ends with rapid-$SAM_CLI_VERSION. False, otherwise
+        """
+
+        rapid_suffix = f"rapid-{SAM_CLI_VERSION}"
```

Comment:
Do we possibly just want to match on a regular expression for `rapid-\d+\.\d+.\d+$` to match any version? In practice it should usually be the SAM CLI version, but if we ever allow some sort of selection scheme this could fail.

Reply:
Updated this part to use regular expression.